src/urllib3/contrib/_securetransport/bindings.py

```diff
@@ -276,6 +276,13 @@
     Security.SSLSetProtocolVersionMax.argtypes = [SSLContextRef, SSLProtocol]
     Security.SSLSetProtocolVersionMax.restype = OSStatus
 
+    try:
+        Security.SSLSetALPNProtocols.argtypes = [SSLContextRef, CFArrayRef]
```

Comment:
The link you posted shows the Swift view of the docs, but In the [ObjectiveC view](https://developer.apple.com/documentation/security/2867327-sslsetalpnprotocols?language=objc) you can see that both arguments are ```Ref```s.
According to [Apple's docs](https://developer.apple.com/documentation/swift/imported_c_and_objective-c_apis/working_with_core_foundation_types) Swift compiler removes the ```Ref``` suffix from CF types since it already manages references by itself. Cool 😃